Extension/package.json

```diff
@@ -293,8 +293,8 @@
         },
         "C_Cpp.intelliSenseCachePath": {
           "type": "string",
-          "default": "${workspaceFolder}/.vscode",
-          "description": "Defines the folder path for cached precompiled headers used by IntelliSense. The default path \"${workspaceFolder}/.vscode\" will be used if a specified path is invalid.",
+          "default": null,
+          "description": "Defines the folder path for cached precompiled headers used by IntelliSense. The default cache path is \"%USERPROFILE%\/.cpptools\" on Windows and \"~/.cpptools\" on Linux and Mac. The default path will be used if no path is specified or if a specified path is invalid.",
```

Comment:
I think we wanted to change this to something more specific like ms-vscode.cpptools.

Reply:
> I think we wanted to change this to something more specific like ms-vscode.cpptools.

I agree. I'll wait for more feedback from the users and then we can decided on the name and final location.

Reply:
I think we're going to go with `%LocalAppData%\Microsoft\vscode-cpptools`